src/main/java/ch/ivyteam/ivy/maven/engine/deploy/http/HttpDeployer.java

```diff
@@ -111,14 +115,21 @@ private HttpEntity getRequestData(File resolvedOptionsFile)
     return builder.build();
   }
 
-  private HttpClientContext getRequestContext(String url) throws URISyntaxException
+  private HttpClientContext getRequestContext(String url) throws URISyntaxException, MojoExecutionException
   {
     String username = "admin";
     String password = "admin";
     if (server != null)
     {
       username = server.getUsername();
-      password = server.getPassword();
+      try
+      {
+        password = secDispatcher.decrypt(server.getPassword());
```

Comment:
do plain passwords still work with this API?
would be happy to see a simple test using this feature ... if not easily possible here: then in docker or a third-party repo (docker-samples)?

Reply:
Yes I've tried it and it worked with plaintext passwords too. Maven encrypted passwords starts and ends with {}.
Even something like this works:
`<password>This is my encrpyed password. {Wdc6XFJMDk4H8L3PspU513LuqxP3QhLcL/8LhAilBaQ=}</password>`

Reply:
But yes as this feature is fully untested we should definitely write some tests somewhere.